README.md

```diff
@@ -1,36 +1,79 @@
-# Iglu Schema Repository
+# Snowplow Iglu
 
-[![Join the chat at https://gitter.im/snowplow/iglu](https://badges.gitter.im/snowplow/iglu.svg)](https://gitter.im/snowplow/iglu?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
-[ ![Build Status][travis-image] ][travis]
-[ ![Release][release-image] ][releases]
-[ ![License][license-image] ][license]
+[![Latest release][latest-release-badge]][latest-release]
+[![License][license-image]][license]
+[![Discourse posts][discourse-image]][discourse]
 
-Iglu is a machine-readable, open-source schema repository for **[JSON Schema][json-schema]** from the team at **[Snowplow Analytics][snowplow-website]**. A schema repository (sometimes called a registry) is like npm or Maven or git, but holds data schemas instead of software or code.
+[![Snowplow logo][logo-image]][website]
 
-Iglu is used extensively in **[Snowplow][snowplow-repo]**. For a presentation on how we came to build Iglu, see **[this blog post][snowplow-schema-post]**.
+## Overview
+
+Iglu is a machine-readable, open-source schema repository for **[JSON Schema][json-schema]** from the team at **[Snowplow Analytics][website]**. A schema repository (also called a registry) is like npm or Maven or git, but holds data schemas instead of software or code.
```

Comment:
I'd suggest dropping "Analytics" from "Snowplow Analytics" since we're heading that way with everything else now.